### ISSUE TITLE
Not clear how to use check command

### ISSUE THREAD

**Issue by s-tikhomirov** [ElementsProject/lightning#3453]
### Issue and Steps to Reproduce

According to [the docs](https://lightning.readthedocs.io/lightning-check.7.html), the `check` command "verifies another command’s parameters without running it". I test the `connect` command as an example here.

I try `connect` without any arguments and expectedly get an error, as `connect` requires an `id` to connect to:
```
$ lightning-cli check connect
{
   "code": -32602,
   "message": "missing required parameter: id"
}
```
But when I provide an `id`, I get:
```
$ lightning-cli check connect 024c2d485bab58bfde4578937bb16160f87cb91868fe2fefc238ea53455e4d7244
{
   "command_to_check": "connect"
}
```
What does this mean? Has the check succeeded?

Moreover, when I try to use a badly formatted `id`, I get the same result:
```
$ lightning-cli check connect qwe
{
   "command_to_check": "connect"
}
```

### `getinfo` output
```
   "id": "02f67d5b7185ac67172807fc8f9d6b000305e93793332ed5a2087478a69909371e",
   "alias": "clightningB",
   "color": "02f67d",
   "num_peers": 2,
   "num_pending_channels": 0,
   "num_active_channels": 2,
   "num_inactive_channels": 0,
   "address": [],
   "binding": [
      {
         "type": "ipv4",
         "address": "127.0.0.1",
         "port": 9740
      }
   ],
   "version": "v0.8.0-40-g899f5de",
   "blockheight": 1664150,
   "network": "testnet",
   "msatoshi_fees_collected": 0,
   "fees_collected_msat": "0msat",
   "lightning-dir": "/home/sergei/.lightningB/testnet"
}
```

**Comment by ZmnSCPxj** [comment]
Yes, if you get `command_to_check` that means checking succeeded.  The `check` is not rigorous: usually only the type of the parameters, and that needed parameters are provided, is checked.  `connect` requires a string as first argument, so the `"qwe"` is considered fine as it is a string; obviously `connect` requires that the string parameter have a particular format, but that is not checkable by `check`.

Basically, if `check` fails, the command will fail with -32602, but if `check` succeeds, it is still possible for the command to still fail with -32602 .  Or, in terms of pure logic, !P -> !Q does not mean P -> Q.

**Comment by s-tikhomirov** [comment]
Would it be worth considering replacing `command_to_check` with a message that more clearly indicates that the check was successful? Possibly with an additional clarification that it does not guarantee successful execution.

**Comment by sangbida** [comment]
This issue is being closed as part of issue cleanup.

**Reason**: Very old issue (6.0 years old) no recent activity

If you believe this issue should remain open, please comment and we can reopen it.
